components/snackbar/snackbar.css

```diff
@@ -0,0 +1,100 @@
+.snackbar {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 99999;
+}
+.snackbar button {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  transition: all 0.2s ease-in-out;
+}
+.snackbar_wrapper {
+  display: none;
+  width: 35rem;
+  height: 4rem;
+  background-color: var(--cta);
+  -webkit-box-shadow: 0 3px 5px -1px rgb(0 0 0 / 20%),
+    0 6px 10px 0 rgb(0 0 0 / 14%), 0 1px 18px 0 rgb(0 0 0 / 12%);
+  box-shadow: 0 3px 5px -1px rgb(0 0 0 / 20%), 0 6px 10px 0 rgb(0 0 0 / 14%),
+    0 1px 18px 0 rgb(0 0 0 / 12%);
+  border-radius: 4px;
+  color: var(--bg-color);
+  justify-content: space-around;
+  font-size: 1.4rem;
+  font-weight: 400;
+  align-items: center;
+}
+
+.sb-baseline {
```

Comment:
Here .sb-baseline and sb-leading are having the same property you can make a separate class for them.

Reply:
It's updated now.